sdk/communication/test-resources.json

```diff
@@ -46,9 +64,21 @@
     }
   ],
   "outputs": {
+    "AZURE_TENANT_ID": {
+      "type": "string",
+      "value": "[parameters('tenantId')]"
+    },
+    "AZURE_CLIENT_ID": {
+      "type": "string",
+      "value": "[parameters('testApplicationId')]"
+    },
+    "AZURE_CLIENT_SECRET": {
+      "type": "string",
+      "value": "[parameters('testApplicationSecret')]"
+    },
     "COMMUNICATION_ENDPOINT": {
       "type": "string",
-      "value": "[concat('https://', parameters('baseName'), '-', parameters('endpointPrefix'), '.communication.azure.com')]"
+      "value": "[concat('https://', parameters('baseName'), '-', parameters('endpointPrefix'), parameters('communicationServicesEndpointSuffix'))]"
```

Comment:
isn't this parameter now removed?

Reply:
communicationServicesEndpointSuffix that is

Reply:
Removed the COMMUNICATION_ENDPOINT